deepchem/models/losses.py

```diff
@@ -1263,7 +1263,7 @@ def loss(substruct_rep, overlapped_node_rep, context_rep,
         return loss
 
 
-class DensityProfileLoss(Loss):
+class XCLoss(Loss):
```

Comment:
Quick reminder to update the name in the docs if you would like to change the name.